### ISSUE TITLE
Property 'turnstile' does not exist on type 'Window & typeof globalThis'

### ISSUE THREAD

**Issue by swiergot** [Le0Developer/turnstile-types#4]
Hi,

It used to work perfectly, but now I have done `yarn upgrade --latest` and I'm getting this:

`error TS2339: Property 'turnstile' does not exist on type 'Window & typeof globalThis'.`

Of course `turnstile-types` is still in `compilerOptions.type` in `tsconfig.json`.

Notable dependencies updated:

@types/node 18.11 -> 20.12
@vitejs/plugin-vue 3.0.3 -> 5.0.4
typescript 4.0.0 -> 5.4.5
vite 3.0.9 -> 5.2.8
vite-plugin-vuetify 1.0.0-alpha.12 -> 2.0.3
vue-tsc 1.0.9 -> 2.0.12

Any ideas?

Thanks.

**Comment by swiergot** [comment]
Ok, I have solved by adding `src/global.d.ts` with:

```
import type { TurnstileObject } from 'turnstile-types'

declare global {
  interface Window { turnstile: TurnstileObject; }
}
```

For some reason it wasn't needed before. Am I missing something or should this be added to README?

**Comment by Le0Developer** [comment]
Looks like our declare global got lost in #3

**Comment by Le0Developer** [comment]
This was fixed in c3e3c354d3c467e2a2b036d2625dfe7dda6a1b4b which is now released as v1.2.1 on NPM.
